src/behavior/features/policyManagement.feature

```diff
@@ -9,7 +9,8 @@ Feature: Create a new policy
     And I can retrieve the created policy for tenant "511" from the Access Control
 
   Scenario Outline: Request validation
-    Given I send a policy creation request to the Access Control for tenant "<tenant>" and subject "<subject>"
+    Given I send a remove request for tenant "511"
```

Comment:
Unfortunately  @before.each_scenario  is not deleting tenant as is expected, I don't know why